everware/spawner.py

```diff
@@ -280,17 +278,19 @@ def start(self, image=None):
         except Exception as e:
```

Comment:
catching all exceptions and then two lines later checking the type of the exception suggests we should catch the TimeoutError separately